### PR TITLE
Redux logger

### PR DESCRIPTION
**Pull request by jczjchiam**
https://www.npmjs.com/package/redux-logger

Useful tool to debug redux flows. Add only for dev environment.

### CODE REVIEW

**Comment by kxmbrian** [comment]
LGTM!

File: client/.eslintrc [hunk 1]
```diff
@@ -10,6 +10,8 @@
     "new-cap": [2, { "capIsNewExceptions": ["Set"] }],
     "react/require-default-props": "off",
     "react/no-array-index-key": "warn",
+    "import/no-extraneous-dependencies": ["warn", { devDependencies: true }],
+    "global-require": "off",
```
Comment (allenwq) [inline client/.eslintrc]: Shall we only disable this in that file? cause we don't want global-require in other places.

Comment (jczjchiam) [inline client/.eslintrc]: Fixed

**Comment by allenwq** [comment]
@jczjchiam Sorry ah, you must rebase and merge your fix with the original commit.  LGTM otherwise 😄

**Comment by jczjchiam** [comment]
Rebased

**Comment by allenwq** [comment]
Thanks @jczjchiam !

**Comment by weiqingtoh** [comment]
Wah :+1: @jczjchiam - Just a quick question, how do we use this? Or is it automatically logged?

**Comment by jczjchiam** [comment]
Do this code change where you set up your redux store.
https://github.com/coursemology-collab/coursemology2/blob/jchiam/redux-logger/client/app/bundles/course/survey/store.js#L7-L9

The logger must be at the end of the middleware pipe, otherwise it'll also log the thunkmiddleware before it.

**Pull request merged**